doc/source/ray-core/fault_tolerance/tasks.rst

```diff
@@ -64,6 +85,28 @@ exception, or pass a list of retryable exceptions. An example is shown below.
   :start-after: __tasks_fault_tolerance_retries_exception_begin__
   :end-before: __tasks_fault_tolerance_retries_exception_end__
 
+
+Use `ray list tasks -f task_id=<task_id>` from :ref:`State API CLI <state-api-overview-ref>` to see task attempts failures and retries:
+
+.. code-block:: bash
+
+  # This API is only available when you download Ray via `pip install "ray[default]"`
```

Comment:
All the comments here is not visible from the documentation. 

<img width="823" alt="Screen Shot 2023-05-30 at 5 57 11 PM" src="https://github.com/ray-project/ray/assets/18510752/2a32ad46-7b18-4edc-ada3-5963f2327e07">

Reply:
Good catch!

Reply:
was missing an empty line.